controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/AbstractRequestProcessor.java

```diff
@@ -166,7 +166,7 @@ public String getProcessorName() {
 
     private <R> CompletableFuture<R> suppressException(CompletableFuture<R> future, R returnOnException, String message) {
         return Futures.exceptionallyExpecting(future, e -> {
-            log.warn(message, e);
+            log.warn(message + " {}", Exceptions.unwrap(e).toString());
```

Comment:
It will be more efficient to do
`log.warn("{} {}", message, Exceptions.unwrap(e).toString())`
or
`log.warn(message + " "+ Exceptions.unwrap(e).toString())`